core/templates/pages/story-editor-page/modal-templates/story-editor-unpublish-modal.component.html

```diff
@@ -1,18 +1,193 @@
-<div class="modal-header">
-  <p>
-    Are you sure you want to unpublish this story? Unpublishing a story will
-    delete any corresponding translation opportunities from the Contributor
-    Dashboard and reject any related pending translation suggestions.
-  </p>
+<div *ngIf="platformFeatureService.status.SerialChapterLaunchCurriculumAdminView.isEnabled">
+  <div class="modal-header">
+    <p>
+      You're attempting to unpublish chapters, please specify a reason and note the warning
+      message before proceeding.
+    </p>
+  </div>
+  <div class="modal-body">
```

Comment:
```suggestion
  <div class="oppia-modal-body">
```

Reply:
Now I have put the prefix in all those classes that are not used by third party. However modal-body class name is used by third party, so I have not changed it.